ocaml/xapi/xapi_vm.ml

```diff
@@ -528,8 +528,12 @@ let clone ~__context ~vm ~new_name =
 (* We do call wait_in_line for snapshot and snapshot_with_quiesce because the locks are taken at *)
```

Comment:
This pre-existing comment suggests that `wait_in_line` is already called. Is that not good enough?